### PR TITLE
Add comment and get comments functionality

### PR DESCRIPTION
**Pull request by jannikschwarz**
Add and get comments

### CODE REVIEW

File: lib/repository.js [hunk 1]
```diff
@@ -41,11 +41,27 @@ async function getUserInformation(uid){
     return userDoc.data();
 }
 
+async function addUserComment(comment){
+    await addDoc(collection(db,'comments/' + comment.movieId + '/movieComments'),comment);
```
Comment (jhviggo) [inline lib/repository.js:45]: You can use string formatting [Like dis](https://www.freecodecamp.org/news/javascript-string-format-how-to-use-string-interpolation-in-js#expressions) if you want. It's a cool feature.
```suggestion
    await addDoc(collection(db, `comments/${comment.movieId}/movieComments`), comment);
```

File: lib/repository.js [hunk 2]
```diff
@@ -41,11 +41,27 @@ async function getUserInformation(uid){
...
+    return comment;
+}
+
+async function getMovieComments(movieId){
+    const snapshot = await getDocs(query(collection(db, 'comments/' + movieId + '/movieComments'), where("movieId", "==", movieId)));
+    const commentsList = [];
+    snapshot.forEach((doc) => {
+        commentsList.push(doc.data());
+    })
+    return commentsList;
```
Comment (jhviggo) [inline lib/repository.js:55]: You can probably do this with the `.map(...)` function if you want to practice some functional programming.

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "node app.js",
     "dev": "nodemon ./app.js",
-    "test": "eslint ./**.js"
+    "test": "mocha --reporter spec"
```
Comment (jhviggo) [inline package.json:10]: This does not belong here. Neither does the `package-lock.json` file.

Comment (jannikschwarz) [reply package.json]: you mean the file itself ?

Comment (jhviggo) [reply package.json]: Yea, the changes to it should probably be reverted. You probably installed some packages for testing mocha and didn't uninstall them correctly, so they're still in the lock file. But it's not the end of the world so I'll approve it anyway.

**Review by jhviggo — APPROVED** [review]

**Pull request merged**